packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -695,6 +747,12 @@ export default {
     }
 }
 
+.c-nav-list--condensed .c-nav-list-item--horizontallyAlignedAboveMid {
```

Comment:
You could group this in the selector above:

```scss
.c-nav-list-item--horizontallyAlignedAboveMid {
    @include f.media('>mid') {
        float: left;
    }
    @include f.media('>wide') {
        margin-left: f.spacing(c);
    }

    .c-nav-list--condensed & {
        @include f.media('>mid', '<huge') {
            margin-left: 0;
        }
    }
}

Reply:
Perfect! I'll get that added too! 😄 Thank you!